app/controllers/batch_updates_controller.rb

```diff
@@ -15,7 +15,8 @@ def create
   def batch_update_params
     defaults = { user_id: current_user.id }
     gene_keys = params[:batch_update][:genes].try(:keys)
-    params.require(:batch_update).permit(artworks: [], genes: gene_keys).merge(defaults)
+    tags = params[:batch_update][:tags]
+    params.require(:batch_update).permit(artworks: [], genes: gene_keys, tags: tags).merge(defaults)
```

Comment:
Let's make sure we sync up on the payload we want to build up. Even though tags are structured differently than genes, we might want to use a parallel kind of structure to handle the adds/removes/ignores in the batch update diff.